charts/merlin/templates/merlin-config-secret.yaml

```diff
@@ -1,5 +1,7 @@
+# Secret is created only if mlp envconfig secret not set
```

Comment:
Can we rename this file to `merlin-environments-secret.yaml`? In retrospect, `merlin-config-secret.yaml` is confusing for me. Is it a configmap or secret?

Reply:
Sure, yep it's a secret. Let's call it `merlin-environments-secret.yaml`